### ISSUE TITLE
Not seeing UV values going into archive table

### ISSUE THREAD

**Issue by earlthesquirrel** [bellrichm/WeeWX-MQTTSubscribe#87]
Trying to isolate where an error is.

I've got a lot of data pushing into weewx.

Most of the data coming in is working fine such as : {"dateTime":1594410217,"outTemp":90.356}

However, the UV readings I'm taking (yes, I added a 2nd UV column...that seems to be in the DB ok)
which look like this  { "dateTime" : 1594410203, "UV" :  0, "UV1" : 0.00 }

aren't populating.   I see the records coming from the sensor and showing up in the MQTT server.
But nothing is making it to the DB.   All the UV and UV1 columns are empty.   I don't see any errors in the syslog.

Any suggestions on how to debug further?

**Comment by bellrichm** [comment]
For me,
Step 1a would be to run from the command line and see what is in the loop packets and archive records. If nothing is in the loop, we know it is MQTTSubscribe.  If it is in the archive, then for some reason WeeWX is not putting it in the DB. Anything else is in the gray area.
Step 1b (it can be done at the same time as 1a) would be to set debug = 1. This will essentially log the inputs and outputs to MQTTSubscribe.

Feel free to attach both. I’m more than happy to take a look.
rich

**Comment by earlthesquirrel** [comment]
Yes that's what I was thinking....

Don't see any UV records in the Loop records even after a cycles.  

I'm pasting from start that went past a couple UV records being posted in MQTT.

LOOP:   2020-07-12 00:59:51 EDT (1594529991) dateTime: 1594529991.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 442.0, soilTemp1: 74.66, usUnits: 1
LOOP:   2020-07-12 00:59:57 EDT (1594529997) altimeter: 1105.760832, dateTime: 1594529997.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 00:59:57 EDT (1594529997) dateTime: 1594529997.0, maxSolarRad: 0.0, outTemp: 76.658, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 00:59:57 EDT (1594529997) dateTime: 1594529997.0, maxSolarRad: 0.0, outHumidity: 67.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 00:59:58 EDT (1594529998) barometer: 28.7447808216, dateTime: 1594529998.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 00:59:58 EDT (1594529998) dateTime: 1594529998.0, maxSolarRad: 0.0, pm1_0: 2090.22, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 00:59:59 EDT (1594529999) c2h5oh: 0.58, dateTime: 1594529999.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 00:59:59 EDT (1594529999) c3h8: 0.65, dateTime: 1594529999.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:00 EDT (1594530000) c4h10: 1.27, dateTime: 1594530000.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:00 EDT (1594530000) ch4: 35.12, dateTime: 1594530000.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:00 EDT (1594530000) co: 2.01, dateTime: 1594530000.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:01 EDT (1594530001) dateTime: 1594530001.0, h2: 0.22, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:01 EDT (1594530001) dateTime: 1594530001.0, maxSolarRad: 0.0, nh3: 0.01, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:02 EDT (1594530002) dateTime: 1594530002.0, maxSolarRad: 0.0, no2: 23.2, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:02 EDT (1594530002) dateTime: 1594530002.0, extraHumid1: 74.4, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:03 EDT (1594530003) dateTime: 1594530003.0, extraTemp1: 75.02, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:01 EDT (1594530001) dateTime: 1594530001.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 442.0, soilTemp1: 74.66, usUnits: 1
LOOP:   2020-07-12 01:00:11 EDT (1594530011) dateTime: 1594530011.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 443.0, soilTemp1: 74.84, usUnits: 1
LOOP:   2020-07-12 01:00:21 EDT (1594530021) dateTime: 1594530021.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 442.0, soilTemp1: 74.66, usUnits: 1
REC:    2020-07-12 01:00:00 EDT (1594530000) altimeter: 1105.760832, barometer: 28.7447808216, c2h5oh: 0.58, c3h8: 0.65, c4h10: 1.27, ch4: 35.12, cloudbase: 3839.55146257, co: 2.01, dateTime: 1594530000, dewpoint: 64.8591735647, ET: None, heatindex: 76.658, humidex: 87.8086895967, interval: 5, maxSolarRad: 0.0, outHumidity: 67.0, outTemp: 76.658, pm1_0: 2090.22, pressure: 27.6268400421, rainRate: 0.0, soilMoist1: 442.0, soilTemp1: 74.66, usUnits: 1
LOOP:   2020-07-12 01:00:34 EDT (1594530034) altimeter: 1105.760832, dateTime: 1594530034.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:35 EDT (1594530035) dateTime: 1594530035.0, maxSolarRad: 0.0, outTemp: 76.658, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:32 EDT (1594530032) dateTime: 1594530032.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 443.0, soilTemp1: 74.66, usUnits: 1
LOOP:   2020-07-12 01:00:35 EDT (1594530035) dateTime: 1594530035.0, maxSolarRad: 0.0, outHumidity: 67.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:35 EDT (1594530035) barometer: 28.7441902219, dateTime: 1594530035.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:36 EDT (1594530036) dateTime: 1594530036.0, maxSolarRad: 0.0, pm1_0: 2090.22, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:36 EDT (1594530036) c2h5oh: 0.58, dateTime: 1594530036.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:37 EDT (1594530037) c3h8: 0.65, dateTime: 1594530037.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:37 EDT (1594530037) c4h10: 1.27, dateTime: 1594530037.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:38 EDT (1594530038) ch4: 35.12, dateTime: 1594530038.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:38 EDT (1594530038) co: 2.01, dateTime: 1594530038.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:39 EDT (1594530039) dateTime: 1594530039.0, h2: 0.22, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:39 EDT (1594530039) dateTime: 1594530039.0, maxSolarRad: 0.0, nh3: 0.01, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:40 EDT (1594530040) dateTime: 1594530040.0, maxSolarRad: 0.0, no2: 23.2, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:40 EDT (1594530040) dateTime: 1594530040.0, extraHumid1: 74.54, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:40 EDT (1594530040) dateTime: 1594530040.0, extraTemp1: 75.092, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:00:42 EDT (1594530042) dateTime: 1594530042.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 444.0, soilTemp1: 74.66, usUnits: 1
LOOP:   2020-07-12 01:00:52 EDT (1594530052) dateTime: 1594530052.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 438.0, soilTemp1: 74.66, usUnits: 1
LOOP:   2020-07-12 01:01:02 EDT (1594530062) dateTime: 1594530062.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 441.0, soilTemp1: 74.66, usUnits: 1
LOOP:   2020-07-12 01:01:12 EDT (1594530072) altimeter: 1105.498368, dateTime: 1594530072.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:01:12 EDT (1594530072) dateTime: 1594530072.0, maxSolarRad: 0.0, outTemp: 76.658, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:01:13 EDT (1594530073) dateTime: 1594530073.0, maxSolarRad: 0.0, outHumidity: 67.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:01:13 EDT (1594530073) barometer: 28.7450761214, dateTime: 1594530073.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:01:14 EDT (1594530074) dateTime: 1594530074.0, maxSolarRad: 0.0, pm1_0: 2090.22, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:01:14 EDT (1594530074) c2h5oh: 0.53, dateTime: 1594530074.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:01:14 EDT (1594530074) c3h8: 0.79, dateTime: 1594530074.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:01:12 EDT (1594530072) dateTime: 1594530072.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 442.0, soilTemp1: 74.66, usUnits: 1
LOOP:   2020-07-12 01:01:15 EDT (1594530075) c4h10: 1.49, dateTime: 1594530075.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:01:15 EDT (1594530075) ch4: 27.66, dateTime: 1594530075.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:01:16 EDT (1594530076) co: 1.88, dateTime: 1594530076.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:01:16 EDT (1594530076) dateTime: 1594530076.0, h2: 0.2, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:01:17 EDT (1594530077) dateTime: 1594530077.0, maxSolarRad: 0.0, nh3: 0.01, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:01:17 EDT (1594530077) dateTime: 1594530077.0, maxSolarRad: 0.0, no2: 24.63, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:01:18 EDT (1594530078) dateTime: 1594530078.0, extraHumid1: 74.12, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:01:19 EDT (1594530079) dateTime: 1594530079.0, extraTemp1: 75.038, maxSolarRad: 0.0, rainRate: 0, usUnits: 1


Next update will be with debug.

**Comment by earlthesquirrel** [comment]
LOOP:   2020-07-12 01:05:53 EDT (1594530353) dateTime: 1594530353.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 445.0, soilTemp1: 74.66, usUnits: 1
LOOP:   2020-07-12 01:06:03 EDT (1594530363) dateTime: 1594530363.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 442.0, soilTemp1: 74.84, usUnits: 1
LOOP:   2020-07-12 01:06:15 EDT (1594530375) altimeter: 1109.172864, dateTime: 1594530375.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:15 EDT (1594530375) dateTime: 1594530375.0, maxSolarRad: 0.0, outTemp: 76.676, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:16 EDT (1594530376) dateTime: 1594530376.0, maxSolarRad: 0.0, outHumidity: 67.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:13 EDT (1594530373) dateTime: 1594530373.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 443.0, soilTemp1: 74.66, usUnits: 1
LOOP:   2020-07-12 01:06:16 EDT (1594530376) barometer: 28.7412372236, dateTime: 1594530376.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:17 EDT (1594530377) dateTime: 1594530377.0, maxSolarRad: 0.0, pm1_0: 1821.23, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:17 EDT (1594530377) c2h5oh: 0.53, dateTime: 1594530377.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:18 EDT (1594530378) c3h8: 0.65, dateTime: 1594530378.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:18 EDT (1594530378) c4h10: 1.27, dateTime: 1594530378.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:19 EDT (1594530379) ch4: 27.66, dateTime: 1594530379.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:19 EDT (1594530379) co: 1.88, dateTime: 1594530379.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:19 EDT (1594530379) dateTime: 1594530379.0, h2: 0.2, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:20 EDT (1594530380) dateTime: 1594530380.0, maxSolarRad: 0.0, nh3: 0.01, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:20 EDT (1594530380) dateTime: 1594530380.0, maxSolarRad: 0.0, no2: 23.89, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:21 EDT (1594530381) dateTime: 1594530381.0, extraHumid1: 74.14, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:21 EDT (1594530381) dateTime: 1594530381.0, extraTemp1: 75.074, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:23 EDT (1594530383) dateTime: 1594530383.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 442.0, soilTemp1: 74.66, usUnits: 1
LOOP:   2020-07-12 01:06:33 EDT (1594530393) dateTime: 1594530393.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 443.0, soilTemp1: 74.84, usUnits: 1
LOOP:   2020-07-12 01:06:43 EDT (1594530403) dateTime: 1594530403.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 442.0, soilTemp1: 74.66, usUnits: 1
LOOP:   2020-07-12 01:06:53 EDT (1594530413) altimeter: 1110.846072, dateTime: 1594530413.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:53 EDT (1594530413) dateTime: 1594530413.0, maxSolarRad: 0.0, outTemp: 76.676, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:54 EDT (1594530414) dateTime: 1594530414.0, maxSolarRad: 0.0, outHumidity: 67.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:54 EDT (1594530414) barometer: 28.7400560243, dateTime: 1594530414.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:54 EDT (1594530414) dateTime: 1594530414.0, maxSolarRad: 0.0, pm1_0: 1821.23, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:55 EDT (1594530415) c2h5oh: 0.51, dateTime: 1594530415.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:55 EDT (1594530415) c3h8: 0.65, dateTime: 1594530415.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:56 EDT (1594530416) c4h10: 1.27, dateTime: 1594530416.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:56 EDT (1594530416) ch4: 24.44, dateTime: 1594530416.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:53 EDT (1594530413) dateTime: 1594530413.0, maxSolarRad: 0.0, rainRate: 0, soilMoist1: 434.0, soilTemp1: 74.66, usUnits: 1
LOOP:   2020-07-12 01:06:57 EDT (1594530417) co: 1.82, dateTime: 1594530417.0, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:57 EDT (1594530417) dateTime: 1594530417.0, h2: 0.19, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:58 EDT (1594530418) dateTime: 1594530418.0, maxSolarRad: 0.0, nh3: 0.01, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:58 EDT (1594530418) dateTime: 1594530418.0, maxSolarRad: 0.0, no2: 24.63, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:59 EDT (1594530419) dateTime: 1594530419.0, extraHumid1: 74.08, maxSolarRad: 0.0, rainRate: 0, usUnits: 1
LOOP:   2020-07-12 01:06:59 EDT (1594530419) dateTime: 1594530419.0, extraTemp1: 75.074, maxSolarRad: 0.0, rainRate: 0, usUnits: 1



Here is (hand edited from scrolling output) message traffic during the same window.

{"dateTime":1594530342,"extraHumid1":74.33}
{"dateTime":1594530344,"extraTemp1":75.074}
{ "dateTime":1594530343, "soilMoist1":444.00, "soilTemp1":74.84 }
{ "dateTime":1594530353, "soilMoist1":445.00, "soilTemp1":74.66 }
{ "dateTime" : 1594530372, "UV" :  0, "UV1" : 0.00}
{"dateTime":1594530375,"altimeter":1109.172864}
{"dateTime":1594530375,"outTemp":76.676}
{"dateTime":1594530376,"outHumidity":67}
{ "dateTime":1594530373, "soilMoist1":443.00, "soilTemp1":74.66 }
{"dateTime":1594530376,"barometer":28.7412372236}
{"dateTime":1594530377,"pm1_0":1821.23}
{"dateTime":1594530377,"c2h5oh":0.53}
{"dateTime":1594530378,"c3h8":0.65}
{"dateTime":1594530378,"c4h10":1.27}
{"dateTime":1594530379,"ch4":27.66}
{"dateTime":1594530379,"co":1.88}
{"dateTime":1594530379,"h2":0.2}
{"dateTime":1594530380,"nh3":0.01}
{"dateTime":1594530380,"no2":23.89}
{"dateTime":1594530381,"extraHumid1":74.14}
{"dateTime":1594530381,"extraTemp1":75.074}
{ "dateTime":1594530383, "soilMoist1":442.00, "soilTemp1":74.66 }
{ "dateTime":1594530393, "soilMoist1":443.00, "soilTemp1":74.84 }
{ "dateTime" : 1594530402, "UV" :  0, "UV1" : 0.00}
{ "dateTime":1594530403, "soilMoist1":442.00, "soilTemp1":74.66 }
{"dateTime":1594530413,"altimeter":1110.846072}
{"dateTime":1594530413,"outTemp":76.676}
{"dateTime":1594530414,"outHumidity":67}
{"dateTime":1594530414,"barometer":28.7400560243}
{"dateTime":1594530414,"pm1_0":1821.23}
{"dateTime":1594530415,"c2h5oh":0.51}
{"dateTime":1594530415,"c3h8":0.65}
{"dateTime":1594530416,"c4h10":1.27}
{"dateTime":1594530416,"ch4":24.44}
{ "dateTime":1594530413, "soilMoist1":434.00, "soilTemp1":74.66 }
{"dateTime":1594530417,"co":1.82}
{"dateTime":1594530417,"h2":0.19}
{"dateTime":1594530418,"nh3":0.01}
{"dateTime":1594530418,"no2":24.63}
{"dateTime":1594530419,"extraHumid1":74.08}
{"dateTime":1594530419,"extraTemp1":75.074}
{ "dateTime":1594530423, "soilMoist1":443.00, "soilTemp1":74.66 }
{ "dateTime" : 1594530432, "UV" :  0, "UV1" : 0.00}
{ "dateTime":1594530433, "soilMoist1":444.00, "soilTemp1":74.66 }
{ "dateTime":1594530443, "soilMoist1":443.00, "soilTemp1":74.66 }
{"dateTime":1594530451,"altimeter":1110.846072}
{"dateTime":1594530451,"outTemp":76.676}
{"dateTime":1594530451,"outHumidity":67}
{"dateTime":1594530452,"barometer":28.7403513241}
{"dateTime":1594530452,"pm1_0":1955.19}
{"dateTime":1594530453,"c2h5oh":0.51}
{"dateTime":1594530453,"c3h8":0.65}
{"dateTime":1594530454,"c4h10":1.27}
{"dateTime":1594530454,"ch4":24.44}
{"dateTime":1594530455,"co":1.82}
{"dateTime":1594530455,"h2":0.19}


With Debug=1 no other output, at least from this window.  I'll let it run for over an archive record and see if I see any output then.  But No Loop records so I suspect it's before the archiving.

**Comment by earlthesquirrel** [comment]
Ok, here's an archive record (please note, I have added the various pollution gas columns as well... they're working fine.

REC:    2020-07-12 01:20:00 EDT (1594531200) altimeter: 1112.213072, barometer: 28.7379397088, c2h5oh: 0.49, c3h8: 0.53, c4h10: 1.07, ch4: 21.5666666667, cloudbase: 3839.31609239, co: 1.76, dateTime: 1594531200, dewpoint: 64.8392091935, ET: None, extraHumid1: 74.2816666667, extraTemp1: 75.044, h2: 0.18, heatindex: 76.637, humidex: 87.7727281042, interval: 5, maxSolarRad: 0.0, nh3: 0.01, no2: 25.1483333333, outHumidity: 67.0, outTemp: 76.637, pm1_0: 4115.51166667, pressure: 27.6202589186, rainRate: 0.0, soilMoist1: 437.64, soilTemp1: 74.84, usUnits: 1

**Comment by bellrichm** [comment]
Setting debug = 1 should add to the log. By default this is usually /var/log/syslog, but depending on your setup it could be other places. At this point all signs are pointing to MQTTSubscribe, so we really need to see that detail.
rich

**Comment by bellrichm** [comment]
I have to ask, since neither UV is showing up, is the topic that they publish to setup correctly for MQTTSubscribe?

**Comment by earlthesquirrel** [comment]
Quick scan of syslog doesn't show any errors.   Good idea to check topic... this is a new sensor so first time for it posting.
Hmmm, the topic has weather/  vs weather on the others.   I'll have to get the sensor and see if that was the culprit.

WIll post after I get it and change the code...

**Comment by bellrichm** [comment]
Sounds like a good plan. 
Just to be clear, I wasn’t looking for errors in the log. It should show what the inputs to MQTTSubscribe are (what it is getting from MQTT) and its outputs (which become loop packets). So if there is no UV data  being logged coming in, something is probably wrong with the MQTT configuration. If there is UV data coming in but not coming out, something is wrong with MQTTSubscribe’s processing.

**Comment by earlthesquirrel** [comment]
Ok, finally got a break to come check it out.
(and yes I got what you were looking for... just wanted to point out no errors as well... )

That was it.  Topic "weather" vs "weather/" (in the past I had this sensor posting to a different system... and the / got left on when I changed the topic.  I didn't realize those two topics were different.  I guess "weather/" is weather with the "empty subtopic"?   In any case, that fixed it.... now getting loop and archive records.

Thanks for the 2nd pair of eyes and suggestions!!
